internal/services/storage/storage_container_resource.go

```diff
@@ -1,26 +1,30 @@
 package storage
 
 import (
-	"fmt"
+	"context"
 	"log"
 	"time"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/tombuildsstuff/giovanni/storage/2018-11-09/blob/containers"
+
 	"github.com/hashicorp/terraform-provider-azurestack/internal/clients"
 	"github.com/hashicorp/terraform-provider-azurestack/internal/services/storage/migration"
 	"github.com/hashicorp/terraform-provider-azurestack/internal/services/storage/parse"
 	"github.com/hashicorp/terraform-provider-azurestack/internal/services/storage/validate"
 	"github.com/hashicorp/terraform-provider-azurestack/internal/tf"
 	"github.com/hashicorp/terraform-provider-azurestack/internal/tf/pluginsdk"
 	"github.com/hashicorp/terraform-provider-azurestack/internal/tf/timeouts"
-	"github.com/tombuildsstuff/giovanni/storage/2018-11-09/blob/containers"
 )
 
-func storageContainer() *schema.Resource {
+func resourceStorageContainer() *schema.Resource {
 	return &schema.Resource{
-		Create: storageContainerCreate,
```

Comment:
Can we return to this kind of functions, even if deprecated? Since azurerm still does not has migrated this and they dont have a solution for timeouts and what not, it is better to follow that instead of changing to context funcs

Reply:
ready!